projects/components/src/table/controls/table-controls.component.ts

```diff
@@ -62,6 +71,19 @@ export class TableControlsComponent implements OnChanges {
   @Input()
   public activeModeItem?: ToggleItem;
 
+  // Checkbox filter
+  @Input()
+  public showCheckbox?: boolean;
```

Comment:
Is this implicit via the other checkbox inputs being provided?

Reply:
Yes. Currently, if checkbox filter model is present and is valid, then the renderer will set this input

Reply:
I was thinking we could move that implicitness down a level - if a label or boolean is provided, then we should show it. Else hide it. No need for the extra input

Reply:
I guess I can make checkboxLabel a required property and hide the filter if it is missing

Reply:
I think that makes sense - we'd never have a checkbox floating without a label (I sure hope...)